ptf/platform_helper/brcm_t0_sai_helper.py

```diff
@@ -262,6 +265,77 @@ def create_host_intf(self):
                 print("Cannot create hostif, error : {}".format(e))
 
 
+    def config_vlan(self):
+        """
+        Default configuation, with Vlan.
+        Craete bridge_port and route interface
+        Steps:
+        """
+        VLAN_ID = 1000
+        PC_PORT_MTU=9100
+        vlan_id = sai_thrift_create_vlan(self.client, vlan_id=VLAN_ID)
+        for i in range(3, 24):
+            bridge_port_id = sai_thrift_create_bridge_port(self.client,
+                    type=SAI_BRIDGE_PORT_TYPE_PORT,
+                    port_id=self.port_list[i],
+                    admin_state=True,
+                    fdb_learning_mode=SAI_BRIDGE_PORT_FDB_LEARNING_MODE_HW)
+        
+            sai_thrift_set_hostif_attribute(self.client, 
+                    hostif_oid=self.hostifs[i], 
+                    vlan_tag=SAI_HOSTIF_VLAN_TAG_KEEP)
+
+            self.vlan_member = sai_thrift_create_vlan_member(
+                    self.client,
+                    vlan_id=vlan_id,
+                    bridge_port_id=bridge_port_id,
+                    vlan_tagging_mode=SAI_VLAN_TAGGING_MODE_UNTAGGED)
+
+            sai_thrift_set_port_attribute(self.client, 
+                    port_oid=self.port_list[i], 
+                    port_vlan_id=VLAN_ID)
+
+        rif_id = sai_thrift_create_router_interface(self.client, 
+            virtual_router_id=self.default_vrf,
+            src_mac_address=ROUTER_MAC,
+            type=SAI_ROUTER_INTERFACE_TYPE_PORT,
+            mtu=PC_PORT_MTU,
+            nat_zone_id=NAT_ZONE_ID)
+            
+        entry = sai_thrift_route_entry_t(
```

Comment:
for creating a direct neighbor, let's loop on route_entry_t and create_route_entry together, with the need ports as condition